Makefile

```diff
@@ -80,10 +80,10 @@ release-install: release-container
 
 verify-shell: docker-running
 	docker run -ti --rm \
-		--privileged \
-		--volume /home/cockpit:/home/user \
+		--privileged --uts=host \
+		--volume /opt/cockpit-verify-secrets:/secrets \
```

Comment:
Moving to ```/var/lib/cockpit-tests```. Docker mounts are ro by default, aren't they? I've made this explicit though to assuage any fears.